src/_pytest/warnings.py

```diff
@@ -123,7 +123,7 @@ def warning_record_to_str(warning_message):
     if unicode_warning:
         warnings.warn(
             "Warning is using unicode non convertible to ascii, "
-            "converting to a safe representation:\n  %s" % msg,
+            "converting to a safe representation:\n  {!r}".format(compat.safe_str(msg)),
```

Comment:
oh, when I suggested {!r}, then you don't need `safe_str` at all I think

Reply:
Oh. Hmm well not sure if this will be a problem as is.

Reply:
No problem, just I was hoping to eventually remove safe_str usages :man_shrugging: